digdag-cli/src/main/java/io/digdag/cli/client/DisableSchedule.java

```diff
@@ -18,7 +18,7 @@
     @Override
     public SystemExitException usage(String error)
     {
-        err.println("Usage: " + programName + " disable <id> | <project-name> [name]");
+        err.println("Usage: " + programName + " disable <schedule-id> | <project-name> [name]");
```

Comment:
trivial: `digdag help` shows

```
    enable <schedule-id>               enable a workflow schedule
```

I aligned with it and `digdag reschedule`.

Reply:
LGTM